src/Common/src/TypeSystem/CodeGen/MethodDesc.CodeGen.cs

```diff
@@ -50,7 +50,7 @@ public virtual bool IsAggressiveInlining
 
         /// <summary>
         /// Gets a value specifying whether the implementation of this method
-        /// is provided by the runtime.
+        /// is provided by the runtime (i.e., through generated IL).
```

Comment:
Nit: We could use "is provided by the runtime" and "is provided by the compiler" as the disambiguator between these two, now that I look at it again.